### PR TITLE
chewing on a toxic root and sucking on a cactus

### PR DESCRIPTION
**Pull request by PlasticCat**
## Description
added datura root, added prickly pear, added pinyon pine, removed space law book from vault

## Motivation and Context
more fallout plants

## How Has This Been Tested?
Complied, it worked

## Changelog (necessary)
:cl:
add: datura root
add: prickly pear
add: pinyon pine
imageadd: added some icons and images: datura sprites, prickly pear sprites, pinyon pine
mapdel: Removed space law book from vault
/:cl:

### CODE REVIEW

**Comment by judgex** [comment]
Let me know if this compiles and I will merge it.

**Comment by PlasticCat** [comment]
unless something changed in code it did compile fine when I made it

**Comment by PlasticCat** [comment]
just tested it, it seems to be fine

**Comment by PlasticCat** [comment]
the conflict is from the removal of the space law book in the vault. I promise it works. I tasted all the plants and all that was removed from the vault was the lore breaking space law book.

**Comment by judgex** [comment]
This now has conflicts due to map updates.  You will need to refresh your fork and rework and resubmit this (some map changes took place).  Sorry for the inconvenience.

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_pie.dm [hunk 1]
```diff
@@ -108,7 +106,7 @@
 	result = /obj/item/reagent_containers/food/snacks/pie/pumpkinpie
 	subcategory = CAT_PIE
 
-/datum/crafting_recipe/food/goldenappletart
+/datum/crafting_recipe/food/*/goldenappletart
```
Comment (Laus4Deus) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_pie.dm:109]: What is this stray `*/`

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_soup.dm [hunk 2]
```diff
@@ -196,7 +194,7 @@
 	subcategory = CAT_SOUP
 
 /datum/crafting_recipe/food/spacylibertyduff
-	name = "Spacy liberty duff"
+	name = "liberty duff"
```
Comment (Laus4Deus) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_soup.dm:197]: **L**iberty duff

File: code/modules/hydroponics/grown/pinyon.dm [hunk 3]
```diff
@@ -1,6 +1,6 @@
-/*/obj/item/seeds/pinyon
+/obj/item/seeds/pinyon
 	name = "pack of pinyon pine seeds"
-	desc = "The seeds of the pinyon pine, known as "pine nuts" or "pi��ns", are an important food for settlers and tribes living in the mountains of the North American Southwest. All species of pine produce edible seeds, but in North America only the pinyon produces seeds large enough to be a major source of food."
```
Comment (Laus4Deus) [inline code/modules/hydroponics/grown/pinyon.dm:3]: In Chile is called Piñón. Leave it n please :(

**Comment by PlasticCat** [comment]
This makes me want to die

**Pull request closed without merging**